settings.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Bierproductie: Info Pages</title>
+    <link rel="stylesheet" href="styles/main.css"/>
+    <script src="scripts/script.js" defer></script>
+    <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+  </head>
+  <body>
+
+    <!-- Side navigation -->
+    <div class="sidenav">
+      <span class="iconify" onclick="location.href='/';" data-icon="cil:touch-app" data-inline="false">
+      </span>
+      <span class="iconify marked-page" onclick="location.href='/settings.html';" data-icon="carbon:settings" data-inline="false">
+      </span>
+    </div>
+
+    <!-- topbar -->
+    <header>       
+      <h2>Bierproductie</h2>
+      <div class="batch">
+        <p id="batch">BATCH</p>
+        <p id="batchNr">3425</p>
+      </div>
+    </header>
+
+    <!-- MAIN PAGE START HERE -->
+    <div class="main">
+      <div class="grid-container">
+        this is the settings page!
```

Comment:
Is this where we can queue batches?

Reply:
that I am not sure of

Reply:
I have made a issue for it here: https://github.com/bierproductie/dashboard/issues/3

Reply:
Great

Reply:
Great.